airflow/kubernetes/pod_generator.py

```diff
@@ -26,7 +26,7 @@
 
 import kubernetes.client.models as k8s
 
-from airflow.executors import Executors
+from airflow.executors import Executors  # pylint: disable=cyclic-import
```

Comment:
?

Reply:
The CI failed on:
```
airflow/utils/log/json_formatter.py:1:0: R0401: Cyclic import (airflow -> airflow.executors -> airflow.executors.kubernetes_executor -> airflow.kubernetes.pod_launcher) (cyclic-import)
airflow/utils/log/json_formatter.py:1:0: R0401: Cyclic import (airflow.executors -> airflow.executors.kubernetes_executor -> airflow.kubernetes.pod_launcher -> airflow.kubernetes.pod_generator) (cyclic-import)
airflow/utils/log/json_formatter.py:1:0: R0401: Cyclic import (airflow.executors -> airflow.executors.kubernetes_executor -> airflow.kubernetes.pod_generator) (cyclic-import)
```

Reply:
Given you otherwise didn't change these files I'm surprised you need it

Reply:
I agree, I got surprised as well, but it kept happening (only in travis, not local).
I can try rebase from master and remove it again, maybe it was a weird travis issue.

Reply:
Ya try rebase against master

Reply:
Removed and rebase, passed local CI, will wait for travis to see.

Reply:
Cool, it passed.